Code/Framework/AzCore/AzCore/IO/Streamer/ReadSplitter.cpp

```diff
@@ -121,10 +121,14 @@ namespace AZ::IO
         auto data = AZStd::get_if<Requests::ReadData>(&request->GetCommand());
         if (data == nullptr)
         {
+            if (auto report = AZStd::get_if<Requests::ReportData>(&request->GetCommand()); report != nullptr)
+            {
+                Report(*report);
+            }
             StreamStackEntry::QueueRequest(request);
             return;
         }
-
+        
```

Comment:
Superfluous whitespace has been added here.

Reply:
Fixed in next commit.